src/plugins/data_view_editor/public/components/empty_prompts/empty_prompts.tsx

```diff
@@ -20,8 +20,7 @@ import { EmptyIndexPatternPrompt } from './empty_index_pattern_prompt';
 import { PromptFooter } from './prompt_footer';
 import { FLEET_ASSETS_TO_IGNORE } from '../../../../data/common';
 
-const removeAliases = (item: MatchedItem) =>
-  !(item as unknown as ResolveIndexResponseItemAlias).indices;
+const removeAliases = (mItem: MatchedItem) => !mItem.item.indices;
```

Comment:
I think this logic was wrong as it never checked inside the item. @mattkime?

Reply:
I'm really not sure off the top of my head. It would be a good idea to create some aliases and verify this is doing whats necessary.

I think this code was removing aliases that didn't reference indices.

Reply:
Verified the use case. The above change is needed as the previous code was not checking inside the item.